src/Schema/SchemaInterface.php

```diff
@@ -9,6 +9,8 @@
 use Digia\GraphQL\Type\Definition\ObjectType;
 use Digia\GraphQL\Type\Definition\TypeInterface;
 
+// TODO: Consider removing this interface.
+
 interface SchemaInterface
```

Comment:
I think we should remove this interface. I don't see why you'd want to replace it. Besides, it's not registered with the container anyway. However, I think I'll do this in a separate PR.

Reply:
If it's not swappable anyways there is no point in having it indeed.